platform-coop-toolkit/coop-map-directory-index
# feat: add contact email field for individuals, add privacy notice (resolves #8)

## Description

This PR adds a new contact email field to the user profile for public display, and adds privacy notices to both individual and organization profile creation flows.

## Steps to test

1. Create/edit/view individual profile with contact email.
2. Create organizational profile.

**Expected behavior:** Both profiles creation forms display an appropriate privacy notice. Email link is added to individual profile. 

## Additional information

Not applicable.

## Related issues

Resolves #8.
